packages/backend-core/src/sql/utils.ts

```diff
@@ -66,6 +66,14 @@ export function buildExternalTableId(datasourceId: string, tableName: string) {
   return `${datasourceId}${DOUBLE_SEPARATOR}${tableName}`
 }
 
+export function checkTableId(tableId: string) {
```

Comment:
Also, this function name is confusing. It does not check the table id, it encodes it